### ISSUE TITLE
 Source Bing Ads - columns missing when compared to official documentation

### ISSUE THREAD

**Issue by SPTKL** [airbytehq/airbyte#14637]
## Environment
- **Airbyte version**:  0.39.35-alpha
- **OS Version / Instance**: Debian GNU/Linux 10 (buster) on GCP
- **Deployment**: Docker
- **Source Connector and version**: [airbyte/source-bing-ads](https://docs.airbyte.io/integrations/sources/bing-ads)	0.1.8
- **Destination Connector and version**: NA
- **Step where error happened**: Sync job

## Current Behavior
Currently the Bing Ads connector is missing a lot of fields that are documented in the official Bing Ads API documentation. e.g. for one of the reports -- Campaign Performance Report, currently the following fields are supported:
```python
report_columns = [
        "AccountName",
        "AccountNumber",
        "AccountId",
        "TimePeriod",
        "CampaignId",
        "CampaignName",
        "DeviceType",
        "Network",
        "Impressions",
        "Clicks",
        "Ctr",
        "AverageCpc",
        "Spend",
        "ReturnOnAdSpend",
        "RevenuePerConversion",
        "ConversionRate",
    ]
```
However, based on [the official documentation](https://docs.microsoft.com/en-us/advertising/reporting-service/campaignperformancereportcolumn?view=bingads-13#syntax) of bing ads for CampaignPerformanceReport, we have way more columns:
```xml
<xs:simpleType name="CampaignPerformanceReportColumn" xmlns:xs="http://www.w3.org/2001/XMLSchema">
  <xs:restriction base="xs:string">
    <xs:enumeration value="AccountName" />
    <xs:enumeration value="AccountNumber" />
    <xs:enumeration value="AccountId" />
    <xs:enumeration value="TimePeriod" />
    <xs:enumeration value="CampaignStatus" />
    <xs:enumeration value="CampaignName" />
    <xs:enumeration value="CampaignId" />
    <xs:enumeration value="CurrencyCode" />
    <xs:enumeration value="AdDistribution" />
    <xs:enumeration value="Impressions" />
    <xs:enumeration value="Clicks" />
    <xs:enumeration value="Ctr" />
    <xs:enumeration value="AverageCpc" />
    <xs:enumeration value="Spend" />
    <xs:enumeration value="AveragePosition" />
    <xs:enumeration value="Conversions" />
    <xs:enumeration value="ConversionRate" />
    <xs:enumeration value="CostPerConversion" />
    <xs:enumeration value="LowQualityClicks" />
    <xs:enumeration value="LowQualityClicksPercent" />
    <xs:enumeration value="LowQualityImpressions" />
    <xs:enumeration value="LowQualityImpressionsPercent" />
    <xs:enumeration value="LowQualityConversions" />
    <xs:enumeration value="LowQualityConversionRate" />
    <xs:enumeration value="DeviceType" />
    <xs:enumeration value="DeviceOS" />
    <xs:enumeration value="ImpressionSharePercent" />
    <xs:enumeration value="ImpressionLostToBudgetPercent" />
    <xs:enumeration value="ImpressionLostToRankAggPercent" />
    <xs:enumeration value="QualityScore" />
    <xs:enumeration value="ExpectedCtr" />
    <xs:enumeration value="AdRelevance" />
    <xs:enumeration value="LandingPageExperience" />
    <xs:enumeration value="HistoricalQualityScore" />
    <xs:enumeration value="HistoricalExpectedCtr" />
    <xs:enumeration value="HistoricalAdRelevance" />
    <xs:enumeration value="HistoricalLandingPageExperience" />
    <xs:enumeration value="PhoneImpressions" />
    <xs:enumeration value="PhoneCalls" />
    <xs:enumeration value="Ptr" />
    <xs:enumeration value="Network" />
    <xs:enumeration value="TopVsOther" />
    <xs:enumeration value="BidMatchType" />
    <xs:enumeration value="DeliveredMatchType" />
    <xs:enumeration value="Assists" />
    <xs:enumeration value="Revenue" />
    <xs:enumeration value="ReturnOnAdSpend" />
    <xs:enumeration value="CostPerAssist" />
    <xs:enumeration value="RevenuePerConversion" />
    <xs:enumeration value="RevenuePerAssist" />
    <xs:enumeration value="TrackingTemplate" />
    <xs:enumeration value="CustomParameters" />
    <xs:enumeration value="AccountStatus" />
    <xs:enumeration value="BudgetName" />
    <xs:enumeration value="BudgetStatus" />
    <xs:enumeration value="BudgetAssociationStatus" />
    <xs:enumeration value="LowQualityGeneralClicks" />
    <xs:enumeration value="LowQualitySophisticatedClicks" />
    <xs:enumeration value="CampaignLabels" />
    <xs:enumeration value="ExactMatchImpressionSharePercent" />
    <xs:enumeration value="CustomerId" />
    <xs:enumeration value="CustomerName" />
    <xs:enumeration value="ClickSharePercent" />
    <xs:enumeration value="AbsoluteTopImpressionSharePercent" />
    <xs:enumeration value="FinalUrlSuffix" />
    <xs:enumeration value="CampaignType" />
    <xs:enumeration value="TopImpressionShareLostToRankPercent" />
    <xs:enumeration value="TopImpressionShareLostToBudgetPercent" />
    <xs:enumeration value="AbsoluteTopImpressionShareLostToRankPercent" />
    <xs:enumeration value="AbsoluteTopImpressionShareLostToBudgetPercent" />
    <xs:enumeration value="TopImpressionSharePercent" />
    <xs:enumeration value="AbsoluteTopImpressionRatePercent" />
    <xs:enumeration value="TopImpressionRatePercent" />
    <xs:enumeration value="BaseCampaignId" />
    <xs:enumeration value="AllConversions" />
    <xs:enumeration value="AllRevenue" />
    <xs:enumeration value="AllConversionRate" />
    <xs:enumeration value="AllCostPerConversion" />
    <xs:enumeration value="AllReturnOnAdSpend" />
    <xs:enumeration value="AllRevenuePerConversion" />
    <xs:enumeration value="ViewThroughConversions" />
    <xs:enumeration value="Goal" />
    <xs:enumeration value="GoalType" />
    <xs:enumeration value="AudienceImpressionSharePercent" />
    <xs:enumeration value="AudienceImpressionLostToRankPercent" />
    <xs:enumeration value="AudienceImpressionLostToBudgetPercent" />
    <xs:enumeration value="RelativeCtr" />
    <xs:enumeration value="AverageCpm" />
    <xs:enumeration value="ConversionsQualified" />
    <xs:enumeration value="LowQualityConversionsQualified" />
    <xs:enumeration value="AllConversionsQualified" />
    <xs:enumeration value="ViewThroughConversionsQualified" />
  </xs:restriction>
</xs:simpleType>
```
## Expected Behavior
All fields should be supported for each table. I should be able to get e.g. `Conversions` fields in all the reports

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?
Yes

**Comment by marcosmarxm** [comment]
Let me know if you need any assistance to submit the PR

**Comment by SPTKL** [comment]
can we close this issue now? since it's already merged

**Comment by marcosmarxm** [comment]
Closed by #14862
